source/base/Debug.ooc

```diff
@@ -34,8 +34,14 @@ Debug: class {
 		This printFunction = f
 	}
 	print: static func (printOut: String, level: Int = 1) {
-		if (This _level == level || (This _level == DebugLevel Everything) ) {
-			This printFunction(printOut)
+		version(debugPrint) {
```

Comment:
No version block here, version block the calls instead using module specific version flags.